sechub-webui/src/main/java/com/mercedesbenz/sechub/webui/StatusController.java

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.webui;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class StatusController {
+    @Autowired
+    StatusService statusService;
+
+    @Value("${sechub.serverurl}")
```

Comment:
In SecHub a RestController does always delegate logic to a service.
I think we should do this same way here. Model logic could be okay inside controller, but not value injection etc.

See also my thoughts about `SecHubServerAccessService` below